sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/AspNetHostedService/src/Azure.Messaging.EventHubs.Processor.HostedService.Samples.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
```

Comment:
Can we please rename to `Azure.Messaging.EventHubs.Processor.Samples.HostedService`?   

The Event Hubs overrides allow for this pattern to also set the `IsSamplesProject` flag that we're using for detecting the build location.  _([src](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Directory.Build.props#L10))_